Source/ACE/Entity/GenericObject.cs

```diff
@@ -15,46 +15,24 @@ public GenericObject(AceObject aceObject)
         {
         }
 
-        public GenericObject(ObjectGuid guid, AceObject aceObject)
-            : base(guid, aceObject)
-        {
-        }
+        ////public GenericObject(ObjectGuid guid, AceObject aceObject)
```

Comment:
```cs

public static WorldObject CreateWorldObject(AceObject aceO)
        {
            WeenieType objWeenieType = (WeenieType?)aceO.WeenieType ?? WeenieType.Generic;

            switch (objWeenieType)
            {
                case WeenieType.LifeStone:
                    return new Lifestone(aceO);
                case WeenieType.Door:
                    return new Door(aceO);
                case WeenieType.Portal:
                    return new Portal(aceO);
                // case WeenieType.PKModifier:
                //    return new PKModifier(aceO);
                case WeenieType.Cow:
                    return new Cow(aceO);
                case WeenieType.Creature:
                    return new Creature(aceO);
                case WeenieType.Container:
                    return new Container(aceO);
                default:
                    return new GenericObject(aceO);
            }
        }

```

Need a default object. I assume this object will eventually have some worth while code in it.